pywr/parameters/parameters.py

```diff
@@ -320,3 +320,34 @@ def __setitem__(self, key, value):
             value = ConstantParameter(value)
         dict.__setitem__(self, key, value)
 
+
+class DiscountFactorParameter(Parameter):
+    """Parameter that returns the current discount factor based on discount rate and a base year.
+
+    Parameters
+    ----------
+    discount_rate : float
+                Discount rate (expressed as 0 - 1) used calculate discount factor for each year.
+        base_year : int
```

Comment:
There's an indenting error here.

Reply:
Thanks, I fixed that should appear in the next commit.